### PR TITLE
Faq component

### PR DESCRIPTION
**Pull request by gitgood-ENT**

### CODE REVIEW

File: stories/frequently-asked-questions.stories.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    { question: 'This is the second question?', answer: 'This is the answer to the second question.' },
+    { question: 'This is the third question?', answer: 'This is the answer to the third question.' },
+    { question: 'This is the fourth question?', answer: 'This is the answer to the fourth question.' },
+    { question: 'This is the fifth question?', answer: 'This is the answer to the fifth question.' },
+]
+
+const Template = args => <FrequentlyAskedQuestions {...args} />
+
+export const FrequentlyAskedQuestionsTest = Template.bind({})
+FrequentlyAskedQuestionsTest.args = {}
```
Comment (ddfridley) [inline stories/frequently-asked-questions.stories.js:22]: To pass the faq as a property to the FAQ component
```
FrequentlyAskedQuestionsTest.args = {faq}
```
Please also create test cases for {faq: []} and {faq: undefined} and make sure the component under test doesn't blow up in either of those cases
 example:
```
export const Empty = Template.bind({})
Empty.args = {}

export const ZeroLength = Template.bind({})
ZeroLength.args = {faq: []}
```

**Review by ddfridley — CHANGES_REQUESTED** [review]
@gitgood-ENT great start, and your code works! Please see the line by line comments.  I explained about animating the height, but that's extra credit.

**Comment by gitgood-ENT** [comment]
All issues resolved, let me know if there are any other changes or questions. Still need to import the have a question component to the bottom once that is completed.

**Review by ddfridley — CHANGES_REQUESTED** [review]
I figure we should just go all in with the height animation. 

I've put in the code for an Opener Component.  It's a common practice to create components to use in other components.  This solves the height problem in a generic way.

**Comment by gitgood-ENT** [comment]
@ddfridley please review changes

**Review by ddfridley — CHANGES_REQUESTED** [review]
Noah - this all look great. The opener looks smooth.  There's just one thing I didn't notice until now.  We need to change the cursor to a pointer when someone hovers over the question of the ^

Look at question.js to see how to do hover in the styles.

Sorry to keep dragging this out - but it's educational right?

**Review by ddfridley — APPROVED** [review]
done.  Thanks!

**Pull request merged**